qiskit_experiments/library/quantum_volume/qv_analysis.py

```diff
@@ -198,19 +196,21 @@ def _calc_quantum_volume(self, heavy_output_prob_exp, depth, trials):
             warnings.warn("Must use at least 100 trials to consider Quantum Volume as successful.")
         if mean_hop > threshold and trials >= 100:
             quantum_volume = 2 ** depth
-            success = True
-
-        result = {
-            "quantum volume": quantum_volume,
-            "qv success": success,
-            "confidence": confidence_level,
-            "heavy output probability": heavy_output_prob_exp,
-            "mean hop": mean_hop,
-            "sigma": sigma_hop,
-            "depth": depth,
-            "trials": trials,
-        }
-        return result
+            quality = "good"
+
+        return AnalysisResultData(
+            "quantum_volume",
+            value=quantum_volume,
```

Comment:
Probably hop? If we want to filter the best qubit group for other experiment, we may want to sort entry by hop value. Current implementation seems like showing QV value though the result is not satisfactory. i.e. If we run QV experiment for qubit 0, 1, 2, 3, 4, 5, this experiment may fail but the entry will be shown as `QV=64, quality="bad"`. This may look like these qubits have QV=64 and confuse the users.

Reply:
Can you give code snippet of what value and name should be here? We can also change this experiment to return multiple analysis results if necesssary.

Reply:
I was thinking to have "HOP" as a name and its value is value. Just wanted to avoid showing bad experiment result with that QV value.